src/scenegraph/LoaderDefinitions.h

```diff
@@ -69,12 +70,36 @@ namespace SceneGraph {
 		bool loop;
 	};
 
+	struct BoundDefinition {
```

Comment:
In general, I see this structure as unnecessarily future-proofed. Do you have any further plans to use `tags[2]` or `params[1]` for other bound types that you plan to implement immediately? If not, I'd recommend reducing the struct to only what's needed to express the minimum set of bounds currently in use.

Reply:
I've got a feeling as most stations are going to be relatively linear, the cylinder / thick line thing is good enough. I'm going to change it to only support that, and if in the future other types are needed an union could be used.

Reply:
Actually using an union doesn't seem like a great idea if strings are used to store the tags, so probably just having different meanings for the tags. For example, an AABB can also be defined by an start and end tag, and the radius could be used for rounding.

AABBs and the cylinders can get you some really complex geometries!